tests_for_fem/test_autodiff_jvp_wrapper.py

```diff
@@ -0,0 +1,115 @@
+"""Testing the ad_wrappers
+1. Check original ad_wrapper against finite differences
+2. Check new ad_wrapper_jvp against finite differences
+    upto second order (time consuming!)
+    Only test reverse mode since it is derived from
+    the specified forward mode
+3. Check that hessian-vector product is not nan
+
+Check_grads reference:
+https://github.com/google/jax/blob/fe30d3fd4bbf92b890c97a75c4c47f4275ab77d1/jax/_src/public_test_util.py#L249
+"""
+import pytest
+import jax
+import jax.numpy as np
+from jax.test_util import check_grads
+from elasticity2d_code import Elasticity
```

Comment:
`from tests_for_fem.elasticity2d_code import Elasticity` is better? Otherwise I cannot run from `jax_am/` with module run - cause an error.